### ISSUE TITLE
Firefox bug: Cursor remains visible in full-screen

### ISSUE THREAD

**Issue by Atemu** [TeamPiped/Piped#3351]
### Official Instance

- [ ] The bug is reproducible on the [official hosted instance](http://piped.video/), or is API-related.

### Describe the bug

When you double tap to fullscreen a video, the cursor remains visible most of the time.

### To Reproduce

1. Use Firefox on macOS
2. Play a video
3. Double-Tap to fullscreen

### Expected behavior

It should hide when the controls hide.

### Browser, and OS with Version.

Firefox 121.0 on macOS Ventura

### Additional context

Sometimes it's hidden as expected but I haven't figured out why.

**Comment by m0d3rnX** [comment]
Same problem on the Linux Flatpak of Firefox

**Comment by FireMasterK** [comment]
I think this might be a Firefox/packaging bug, since I can't reproduce this on the Firefox package from Arch Linux, but I can reproduce this on the Flatpak version.

**Comment by Atemu** [comment]
I can repro this on NixOS aswell. Firefox on macOS is the official binary by Mozilla.

**Comment by ClemaX** [comment]
Running into this issue on LibreWolf too (`librewolf` package from the AUR).

**Comment by Atemu** [comment]
One thing of note might be that autoplay is disabled for me; in the settings aswell as the browser.

**Comment by Bnyro** [comment]
> Running into this issue on LibreWolf too (`librewolf` package from the AUR). Maybe relevant firefox bugtrackker issue: [CSS `cursor:none` not working ](https://bugzilla.mozilla.org/show_bug.cgi?id=1876269)
>
I agree, this looks like a recently introduced Firefox issue, as we didn't touch that player code recently and it hasn't been an issue before.

**Comment by Kokofruits1** [comment]
The cursor hides if it is right under the seek bar, everywhere else it doesn't. I have no idea if it is useful in any way, but it is kind of a workaround for now.

Fedora KDE 39 (wayland), Firefox 123.0.1

**Comment by Atemu** [comment]
Another workaround is to press the space bar to play the video (or double tab if it's already playing).
